contrib/megatools/update.py

```diff
@@ -1 +1 @@
-pattern = r"megatools-(\d+.\d+.\d+)."
+pattern = r"megatools-(\d+.\d+.\d+.\d+)."
```

Comment:
seems now you can remove this entirely, the defaults work after deleting it:  

```
$ ./cbuild update-check contrib/megatools s              
Checking for updates: megatools=1.11.1.20230212
Adding 'https://megatools.megous.com' for version check...
Adding 'https://megatools.megous.com/builds/' for version check...
Fetching 'https://megatools.megous.com' for version checks...
Fetching 'https://megatools.megous.com/builds/' for version checks...
Checking found version: 1.10.2
Checking found version: 1.10.3
Checking found version: 1.11.0.20220519
Checking found version: 1.11.1.20230212
```

Reply:
ah, nice. Seems like I made more work for myself for no reason :)